deploy/olm-catalog/smart-gateway-operator/manifests/smart-gateway-operator.clusterserviceversion.yaml

```diff
@@ -55,83 +82,23 @@ spec:
       kind: SmartGateway
       name: smartgateways.smartgateway.infra.watch
       resources:
-      - kind: Pod
-        name: ""
-        version: v1
-      - kind: Deployments
-        name: ""
-        version: v1
-      - kind: ReplicaSets
-        name: ""
-        version: v1
       - kind: Services
         name: ""
         version: v1
-      - kind: ConfigMaps
-        name: ""
-        version: v1
       - kind: SmartGateways
         name: smartgateways.smartgateway.infra.watch
-        version: v2alpha1
+        version: v2
       - kind: ServiceMonitors
         name: servicemonitors.monitoring.coreos.com
         version: v1
```

Comment:
Found out this section is really just informational, and is intended to point the administrator at components that make sense to look in to when debugging. You shouldn't list things like `ConfigMaps` etc that are managed for you and not intended to be modified by the administrator. This this is thus reduced to better meet that description. See https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#your-custom-resource-definitions